### ISSUE TITLE
This project could use some love...

### ISSUE THREAD

**Issue by jlk** [apex/gateway#45]
I'm looking around for a lambda adapter to use with [go-chi/chi](https://github.com/go-chi/chi)...Apex Gateway looks neat, but...unloved (trying to avoid calling this "abandoned").

* Issues left open when they could be closed
* Unanswered questions in issues
* PRs sitting open for months

I know one of the maintainers left $employer, but is this of interest to Apex to maintain, or maybe add some outside maintainers? Is there a popular fork of this, perhaps that I'm not aware of?

[awslabs/aws-lambda-go-api-proxy](https://github.com/awslabs/aws-lambda-go-api-proxy) is equally un-loved (including security vulns), [davyzhang/agw](https://github.com/davyzhang/agw) is interesting but lacking stars. Is there some other well-maintained way that people are integrating lambda and go http routers?

**Comment by earthboundkid** [comment]
I have a fork, https://github.com/carlmjohnson/gateway. I am actively using my fork, so if there’s a bug in it, I’d like to know. I’m not working on new features, however.

**Comment by jlk** [comment]
@carlmjohnson looks like you're a few commits behind this project...

> This branch is 15 commits ahead, 10 commits behind apex:master.

**Comment by earthboundkid** [comment]
No, those commits are all for features I’m deliberately not taking.
